doc/user/content/sql/isolation-level.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Isolation Level"
+description: "The isolation level defines how Materialize isolates the execution of transactions."
+aliases:
+ - /sql/consistency
+menu:
+  main:
+    parent: 'reference'
+    weight: 160
+---
+
+The SQL standard defines four levels of transaction isolation. In order of least strict to most strict they are Read
+Uncommitted, Read Committed, Repeatable Read, and Serializable. In Materialize, you can request any of these isolation
+levels, but they all behave the same as the Serializable isolation level. In addition to the four levels defined in the
+SQL Standard, Materialize also defines a Strict Serializable isolation level.
+
+Isolation level is a per session configurable variable. To set the current session’s isolation level you can
+execute `SET TRANSACTION_ISOLATION TO '<isolation-level>';`. The default isolation level is Strict Serializable.
```

Comment:
Are the `<isolation-level>` values the same as mentioned before?

Read Uncommitted
Read Committed
...

Reply:
The best way would be to add syntax like the other commands ([e.g.](https://materialize.com/docs/sql/create-role/)). But that's a task for the issue https://github.com/MaterializeInc/materialize/issues/10218.

I would suggest an example.

Reply:
I just added some documentation that includes syntax and valid values. Once #10218 is finished we can swap it out with a link. I'll post a comment to that issue as well.